Pcap++/header/PcapLiveDevice.h

```diff
@@ -226,13 +232,14 @@ namespace pcpp
 			 * to capture all the data available from the packet.
 			*/
 			explicit DeviceConfiguration(DeviceMode mode = Promiscuous, int packetBufferTimeoutMs = 0, int packetBufferSize = 0,
-				                PcapDirection direction = PCPP_INOUT, int snapshotLength = 0)
+				                PcapDirection direction = PCPP_INOUT, int snapshotLength = 0, unsigned int extra = 0)
```

Comment:
Doxy gives error since there is an undocumented new parameter, the documentation of the function should be also updated.